### PR TITLE
Add ChatQnA E2E test workflow

### PR DESCRIPTION
**Pull request by XuehaoSun**
## Type of Change

feature or bug fix or documentation or validation or others

## Description

detail description

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: ChatQnA/tests/test_chatqna_on_xeon.sh [hunk 1]
```diff
@@ -143,6 +168,8 @@ function main() {
     validate_microservices
     validate_megaservice
 
+    run_e2e_tests
```
Comment (chensuyue) [inline ChatQnA/tests/test_chatqna_on_xeon.sh:171]: use `validate_ui` or `validate_frontend`.  
`e2e` including backend and frontend.

Comment (XuehaoSun) [reply ChatQnA/tests/test_chatqna_on_xeon.sh]: fixed

**Comment by XuehaoSun** [comment]
@WenjiaoYue please check E2E UI test [timeout issue](https://github.com/opea-project/GenAIExamples/actions/runs/9121621925/job/25081093135?pr=144#step:4:1543)

**Comment by WenjiaoYue** [comment]
> @WenjiaoYue please check E2E UI test [timeout issue](https://github.com/opea-project/GenAIExamples/actions/runs/9121621925/job/25081093135?pr=144#step:4:1543)

The back-end chat response speed needs to be extended. The PR has been modified and submitted.

**Review by chensuyue — APPROVED** [review]

**Review by ftian1 — APPROVED** [review]

**Pull request merged**